app/models/expression_file_info.rb

```diff
@@ -14,29 +14,43 @@ class ExpressionFileInfo
   UNITS_VALUES = [nil, 'UMI-corrected raw counts', 'raw counts']
   validates :units, inclusion: {in: UNITS_VALUES}
 
-  BIOSAMPLE_INPUT_TYPE_VALUES = [nil, 'whole cell', 'single nuclei', 'bulk']
+  BIOSAMPLE_INPUT_TYPE_VALUES = [nil, 'Whole cell', 'Single nuclei', 'Bulk']
```

Comment:
These changes require the corresponding values for the synthetic studies also need updating:
db/seed/synthetic_studies/human_raw_counts/study_info.json#L21
db/seed/synthetic_studies/chicken_raw_counts/study_info.json#L21